codes/rust/array_queue.rs

```diff
@@ -0,0 +1,100 @@
+/**

```

Comment:
The location of this file should in "codes/rust/chapter_stack_and_queue/array_queue.rs"